bin/deploy.sh

```diff
@@ -23,10 +25,13 @@ export DRONE_SOURCE_BRANCH=$(echo $DRONE_SOURCE_BRANCH | tr '[:upper:]' '[:lower
 
 if [[ ${KUBE_NAMESPACE} == ${BRANCH_ENV} ]]; then
   $kd -f kube/configmaps -f kube/certs
-  $kd -f kube/redis -f kube/app
+  $kd -f kube/redis -f kube/app -f kube/file-vault
```

Comment:
I would suggest moving `kube/file-vault` before `kube/app`, this will set the dependency of redis and file-vault services, and app won't be deployed if there is a failure in the dependency.